.github/workflows/pr-check.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python: ['3.7', '3.8', '3.9', '3.10', '3.11-dev']
```

Comment:
whats the difference beteen 3.11-dev and regular 3.11? It's saying: "Header files and a static library for Python" when I search up, but still not really sure what that means

Reply:
It means latest development version. Usually we turn this on for the next version of python. we should be setting 3.12-dev soon.